### DISCUSSION TITLE
Source of documentation?

### DISCUSSION THREAD

**Discussion by thecashewtrader** [themesberg/flowbite#302] [Q&A]
I want to contribute to the docs, but I can't seem to find the source. Could anyone point me to it?

**Comment by zoltanszogyenyi** [comment]
Hello @thecashewtrader,

You can contribute by cloning the whole repo: https://github.com/themesberg/flowbite

You need to have HUGO installed and then run `npm install` and `npm run start`.

**Reply by thecashewtrader** [reply]
Thanks a lot! Also, any idea where the docs for the "integrations" (eg with React, SvelteKit etc.) are?
